buildpack.md

```diff
@@ -91,6 +80,54 @@ Buildpack API versions:
  - `<major>` and `<minor>` MUST only contain numbers (unsigned 64 bit integer)
  - When `<major>` is greater than `0` increments to `<minor>` SHALL exclusively indicate additive changes
 
+## Terminology
+
+### CNB Terminology
+
+A **buildpack** is a directory containing a `buildpack.toml`. A buildpack analyzes application source code and determines the best way to build it.
+
+A **buildpack group**, or **group**, is a list of one or more buildpacks that are designed to work together - for example, a buildpack that provides `node` and a buildpack that provides `npm`.
+
+An **order** is a list of one or more groups to be tested against application source code, so that the appropriate group for a build can be determined. 
+
+An **executable buildpack** is a buildpack containing `/bin/detect` and `/bin/build` executables.
```

Comment:
```suggestion
A **component buildpack** is a buildpack containing `/bin/detect` and `/bin/build` executables.
```
Feedback from 4/21 Working Group: `executable buildpack` is a little misleading because composite buildpacks are still "executable" in the sense that their components are executable.